src/main.c

```diff
@@ -19,6 +19,14 @@
 
 int main(int argc, char **argv)
 {
+#ifdef __sparc
```

Comment:
I would like to see this else where, like in util-cpu.c/h and then be a single call from main